tests/unit/network_graph/test_graph.py

```diff
@@ -284,39 +275,161 @@ def test_capacity_path_multi_hops_positive_balance(complex_community_with_trustl
     assert value == 60000
 
 
+def test_capacity_path_single_hop_with_fees(complex_community_with_trustlines_and_fees):
```

Comment:
The following tests are all the same only with different data. This is not acceptable, this is so much code that does not add anything but makes more work to review, and to maintain. We have to cut that down by reusing the code only feeding it with the different test data.

Reply:
I merged some of them